Index: src/main/java/org/example/base/MyBot.java
===================================================================
diff --git a/src/main/java/org/example/base/MyBot.java b/src/main/java/org/example/base/MyBot.java
--- a/src/main/java/org/example/base/MyBot.java	
+++ b/src/main/java/org/example/base/MyBot.java	
@@ -44,7 +44,8 @@
 
                 } else if (text.equals(BaseBot.BUYURTMA)) {
                     myExecute(replyKeyboardMarkup(List.of(BaseBot.PLACE,BaseBot.FOODS), 1), null, "Tanlovni amalga oshiring", chatId);
-                } else if (text.equals(BaseBot.BUYURTMALARIM)) {
+                } else if (text.equals(BaseBot.PLACE)) {
+
 
                 }
             }
Index: src/main/java/org/example/model/User.java
===================================================================
diff --git a/src/main/java/org/example/model/User.java b/src/main/java/org/example/model/User.java
--- a/src/main/java/org/example/model/User.java	
+++ b/src/main/java/org/example/model/User.java	
@@ -5,6 +5,7 @@
 public class User {
     private long chatId;
     private String step;
-    private String name;
-    private List<Product> products;
+    private String msg;
+    private List<Product> productList;
+
 }
Index: src/main/java/org/example/model/Product.java
===================================================================
diff --git a/src/main/java/org/example/model/Product.java b/src/main/java/org/example/model/Product.java
--- a/src/main/java/org/example/model/Product.java	
+++ b/src/main/java/org/example/model/Product.java	
@@ -1,7 +1,17 @@
 package org.example.model;
 
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.Setter;
+
+@Getter
+@Setter
+@AllArgsConstructor
 public class Product {
+    private int categoryId;
+    private int id;
     private  String name;
     private double price;
     private int amount;
+
 }
